templates/todo/projects/java-mongo/.repo/bicep/infra/main.bicep

```diff
@@ -35,6 +35,8 @@ var abbrs = loadJsonContent('../../../../../../common/infra/bicep/abbreviations.
 var resourceToken = toLower(uniqueString(subscription().id, environmentName, location))
 var tags = { 'azd-env-name': environmentName }
 
+var webName = !empty(webServiceName) ? webServiceName : '${abbrs.webSitesAppService}web-${resourceToken}'
```

Comment:
Why var here in this template and not in others?

Reply:
Done